actions/access_keys/count.go

```diff
@@ -14,15 +14,14 @@ import (
 // @Description	Count of access keys
 // @Tags		Access-key
 // @Produce		json
-// @Param		metadata query string false "metadata"
-// @Param		conditions query string false "conditions"
+// @Param		CountRequestParameters body actions.CountRequestParameters false "CountRequestParameters model containing metadata and conditions"
 // @Success		200
```

Comment:
We should add also a model of the response to all endpoints.

Reply:
Done